jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/commands/ExecutionErrorCleanupCommand.java

```diff
@@ -115,7 +115,7 @@ public ExecutionResults execute(CommandContext ctx) throws Exception {
 		Map<String, Object> parameters = new HashMap<>();
         StringBuilder cleanUpErrorsQuery = new StringBuilder();
         
-        cleanUpErrorsQuery.append("delete from ExecutionErrorInfo where processInstanceId in "
+        cleanUpErrorsQuery.append("delete from ExecutionErrorInfo where PROCESS_INST_ID in "
```

Comment:
Hi, the string here is JPQL (Java Persistence Query Language), not SQL. JPQL operates on Java entities which means that the identifier here should be the name of the Java field:

https://github.com/kiegroup/jbpm/blob/cb8c01be15c79524c0e3218db0f082e67506d404/jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/jpa/ExecutionErrorInfo.java#L94-L98

So the query here is correct.